services/blockchain-indexer/README.md

```diff
@@ -58,7 +58,10 @@ A list of the most commonly used environment variables is presented below:
 - `JOB_SCHEDULE_REFRESH_ACCOUNT_KNOWLEDGE`: Job run cron schedule to refresh account knowledge. By default, it is set to run every 15th minute (`*/15 * * * *`).
 - `JOB_INTERVAL_DELETE_FINALIZED_CCU_METADATA`: Job run interval to delete finalized CCU metadata. By default, it is set to 0.
 - `JOB_SCHEDULE_DELETE_FINALIZED_CCU_METADATA`: Job run cron schedule to delete finalized CCU metadata. By default, it is set to run once a day at 02:00 am (`0 2 * * *`).
+- `JOB_INTERVAL_TRIGGER_ACCOUNT_UPDATES`: Job run interval to trigger account updates. By default, it is set to 0.
+- `JOB_SCHEDULE_TRIGGER_ACCOUNT_UPDATES`: Job run cron schedule to trigger account updates. By default, it is set to run every 15th minute (`*/15 * * * *`).
```

Comment:
```suggestion
- `JOB_SCHEDULE_TRIGGER_ACCOUNT_UPDATES`: Job run cron schedule to trigger account updates. By default, it is set to run every 15 minutes (`*/15 * * * *`).
```

Reply:
I think, `15th minute` here is correct since the job runs exactly at xx:00, xx:15, xx:30 and xx:45 hrs.

Reply:
Since it's an interval, we should define the duration of the interval, such as "after every 15 minutes". To me, 15 minutes sounds better.